src/test/java/org/myrobotlab/service/ArduinoTest.java

```diff
@@ -483,15 +453,13 @@ public final void testServo() throws Exception {
     // notify and process releasing itself from attached
     // services
     servo.releaseService();
-
+    Service.sleep(200); // async call removal of servo
```

Comment:
feels like a race condition here...

Reply:
Arduino is a bit tough ... its attached in two realms java-land and mrlcom land
and they have to be sync'd
and there's connections and serial and ... and...
yeah still too much complexity ... but the "Java land is always right, mrlcomm needs to follow" is certainly helps
(but there are still details .... that are not all resolved)